modules/print.py

```
@@ -21,7 +21,10 @@ def call(self, data: AkariData, params: Any, callback: AkariModuleType | None =
         self._logger.debug("Data: %s", data)
         self._logger.debug("Params: %s", params)
 
-        self._logger.debug("Last Data: %s", json.dumps(data.last(), indent=4))
+        try:
+            self._logger.debug("Last Data: %s", json.dumps(data.last(), indent=4))
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)